xrpl4j-crypto-parent/xrpl4j-crypto-core/pom.xml

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xmlns="http://maven.apache.org/POM/4.0.0"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  xmlns="http://maven.apache.org/POM/4.0.0"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <parent>
     <groupId>org.xrpl</groupId>
     <artifactId>xrpl4j-crypto-parent</artifactId>
-    <version>HEAD-SNAPSHOT</version>
+    <version>v3-dev-preview-SNAPSHOT</version>
```

Comment:
Note: These should be `HEAD-SNAPSHOT` before we merge this PR.

Reply:
This is done [here](https://github.com/XRPLF/xrpl4j/pull/348/commits/d23308232e37248cc3f7af7b982dd91e744c9c57)